modules/metering-reports.adoc

```diff
@@ -261,8 +261,12 @@ spec:
 [id="metering-runImmediately_{context}"]
 == runImmediately
 
-Set `spec.runImmediately` to `true` to run the report immediately with all available data, regardless of the `reportingStart` or `reportingEnd` values, and without checking if there is any data for the report period.
-For reports with a schedule set, it will not wait for each period's reportingEnd to elapse before processing and all reportPeriods between `reportingStart` and `reportingEnd`.
+When `runImmediately` is set to `true`, the report will be run immediately. This is a simple way to have a report get processed and queued right away without setting schedule values that could cause it to get requeued.
```

Comment:
s/will be run/runs

To avoid some tense and modal issues, I might replace the second sentence:
> This behavior ensures that the report is immediately processed and queued without requiring additional scheduling parameters.